### PR TITLE
slider for Ethereum transaction

### PR DESCRIPTION
**Pull request by valpinkman**
Adds a slider to select the gas of a transaction for Ethereum-like currencies

### Type

Feat

### Context

LL-1363

### Parts of the app affected / Test plan

Send Flow => step 4 Summary => edit gas

IMG_9701E8CB6E84-1

### CODE REVIEW

File: src/families/ethereum/EditFeeUnitEthereum.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      onValueChange={setValueIndex}
+      minimumValue={0}
+      maximumValue={range.steps - 1}
+      thumbTintColor={colors.live}
+      minimumTrackTintColor={colors.live}
+    />
+  );
+});
+
+const fallbackGasPrice = BigNumber(10000000000);
```
Comment (gre) [inline src/families/ethereum/EditFeeUnitEthereum.js]: probably not needed

File: src/families/ethereum/EditFeeUnitEthereum.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  const [gasPrice, setGasPrice] = useState(() =>
+    bridge.getTransactionExtra(mainAccount, transaction, "gasPrice"),
+  );
+  const feeCustomUnit = bridge.getTransactionExtra(
+    mainAccount,
+    transaction,
+    "feeCustomUnit",
+  );
+
+  const fees = bridge.getTransactionNetworkInfo(mainAccount, transaction);
```
Comment (gre) [inline src/families/ethereum/EditFeeUnitEthereum.js]: this is guaranteed to be filled in mobile paradigm. You can assume it's set. so feel free to `if (!fees) return null` after this line so you don't need a `fallbackGasPrice`

File: src/locales/en/common.json [hunk 3]
```diff
@@ -878,6 +880,8 @@
       "title": "Network fees",
       "validate": "Confirm",
       "required": "Fees are required",
+      "chooseGas": "Choose a gas price",
+      "higherFaster": "Higher gas price means a faster confirmation",
```
Comment (gre) [inline src/locales/en/common.json]: @dasilvarosa please confirm. thanks! (maybe you already did on slack, just to be sure :p )

Comment (valpinkman) [reply src/locales/en/common.json]: on slack it seems that the wording did come from @dasilvarosa

**Comment by gre** [comment]
to be tested that all custom summary edit screen still works in all of the 3 cases: ETH, BTC, XRP (edit fee, edit tag for xrp)

**Comment by valpinkman** [comment]
I did test for BTC (did not have any XRP) and it seems to still be okay

**Pull request merged**